embeddings_test.go

```diff
@@ -116,3 +126,1606 @@ func TestEmbeddingEndpoint(t *testing.T) {
+					},
+				},
+			},
+			want: EmbeddingResponse{
+				Data: []Embedding{
+					{
+						Embedding: []float32{
+
+							-0.006968617,
+							-0.0052718227,
+							0.011901081,
+							-0.024984878,
+							-0.024554798,
```

Comment:
Hey, thank you for your work here! It seems that the embedding size is not fixed anywhere in the library, so maybe for test sake we can make them a bit smaller, say 3 or 4 dimensional?